test/cpp/microbenchmarks/bm_callback_cq.cc

```diff
@@ -0,0 +1,226 @@
+/*
+ *
+ * Copyright 2015 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/* This benchmark exists to ensure that immediately-firing alarms are fast */
+
+#include <benchmark/benchmark.h>
+#include <grpc/grpc.h>
+#include <grpcpp/alarm.h>
```

Comment:
Check which header files you actually need. I think many here are not needed such as alarm or C++ completion queue.